### ISSUE TITLE
Notes lost on upgrade (iPhone).

### ISSUE THREAD

**Issue by DiagonalArg** [standardnotes/forum#2239]
iPhone SE, gen.1, iOS 15.7

StandardNotes v. 3.44.3 (281)

On upgrade to newest version, I opened the app and found no notes.  I was still logged in, so that scared me.  I turned off wifi and opened the web app from my laptop browser.  At first I was releived to find my notes present, but it turned out they were all 1 week old.  (Of course I didn't have a backup.)

I logged out on the phone, reconnected to wifi, and logged back in.  The one week old notes propogated to the phone.

**Comment by effieeee** [comment]
Hi @DiagonalArg, I'm sorry to hear that, may I first ask if you are also unable to see your complete set of notes when you sign into the [web app](https://app.standardnotes.com/)?

If not, can you send us another message at help@standardnotes.com using your Standard Notes account email, for verification purposes?

**Comment by DiagonalArg** [comment]
That's right.  As I explained, when I logged into the web app, I could see my notes, but they were all one week old.

I'll send you an email with a link to this issue.  Thank you.

**Comment by effieeee** [comment]
Thanks for sending us the email! I consulted with someone on the team regarding your issue as well, and we just wanted to ask what device or app platform you were using the most to edit your notes during the past week? We're worried that your notes might not have been syncing properly on that device for some reason, which is why they're appearing a week old when you now sign in 😞

**Comment by DiagonalArg** [comment]
I used the iphone all week.

If the device is not syncing properly, then that's a *second* problem.  The first problem is that there were no notes at all on the phone, after the app update.
